package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@zowe/zos-ftp",
-  "version": "0.1.0",
+  "name": "@brightside/zos-ftp-for-zowe-cli",
+  "version": "1.0.0",
```

Comment:
Before going to a 1.0.0 I would like to run the CLI commands by some UX people (unless you already did this) : )

Reply:
I change it back to `0.1.0`.